tests.py

```diff
@@ -284,13 +297,15 @@ def test_get_sld_backward_compatibility_sld_for_empty_string(self):
 
     def test_get_sld_backward_compatibility_sld_for_fqdn(self):
         psl = publicsuffix.PublicSuffixList()
-        assert 'foo.com' == psl.get_sld('www.foo.com.')
+        # 'www.foo.com.' -> <www>.<foo>.<com>.<empty> -> None, empty labels are not allowed
+        assert None == psl.get_sld('www.foo.com.')
```

Comment:
same question abut strip

i don't see any tests for the unsafe methods. shouldn't there be a few? this is what the big change is.